README.md

```diff
@@ -120,7 +120,27 @@ $ run-full-environment
 ```
 
 It is also possible to only run Nginx in that shell with `run-nginx`. This is
-usefull for instance to proxy a `cty serve` running from GHCi.
+useful for instance to proxy a `cty serve` running from GHCi.
+
+The `run-vm-test` attribute let's you run a NixOS VM integration test
+that mirrors the production setup. You can run it either interactively
+or non-interactively:
+
+```sh
+$ nix-build -A run-vm-test
+# ^ Will run the VM test in a non interactive fashion.
+$ nix-build -A run-vm-test.driverInteractive && ./result/bin/nixos-test-driver
```

Comment:
I've run this build after `run-vm-test` and it seems it's downloading a specific version of QEMU:

```
these 2 paths will be fetched (150.54 MiB download, 1060.83 MiB unpacked):
  /nix/store/chzhy699v0nd72xgnh1hh94qxl0ikdkk-qemu-7.0.0-ga
  /nix/store/zmiwnjbx63q1gm2dx86zlx08cgz81p0q-qemu-7.0.0
```
Any idea why (instead of using the same as the first build attribute) ?

Reply:
Yes.

The non interactive test uses a qemu stripped from its interactive-specific dependencies. Notably qtk, alsa, pulse audio, vnc, etc. 

The idea is to keep the NixOS non interactive runner as stripped as possible. You can see the details here: https://github.com/NinjaTrappeur/nixpkgs/blob/1a6a4817e8264529e1e20fd47b88a19c9bfe03b4/pkgs/applications/virtualization/qemu/default.nix#L9 `nixosTestRunner` is used to disable the interactive deps.

You need these dependencies for a nice interactive experience though.

Reply:
Thanks.